### PR TITLE
feat: dev environment backend setup

### PR DESCRIPTION
**Pull request by EmperorOrokuSaki**

### CODE REVIEW

**Review by zoruka — APPROVED** [review]

File: contracts/deployments/qanet/FleekERC721.json
Comment (zoruka) [inline contracts/deployments/qanet/FleekERC721.json:1]: do we need the deployment files from qanet to be pushed with the repo code?

File: serverless/src/functions/apps/handler.ts [hunk 1]
```diff
@@ -75,7 +71,11 @@ export const submitAppInfo = async (
   try {
     // Check the parameters and environment variables
     dotenv.config();
-    if (event.body === null || process.env.BUNNY_CDN_ACCESS_KEY == undefined || event.headers.originUrl === undefined) {
+    if (
+      event.body === null ||
+      process.env.BUNNY_CDN_ACCESS_KEY == undefined ||
```
Comment (zoruka) [inline serverless/src/functions/apps/handler.ts:76]:
```suggestion
      process.env.BUNNY_CDN_ACCESS_KEY === undefined ||
```

**Review by zoruka — APPROVED** [review]

File: contracts/deployments/qanet/FleekERC721.json
Comment (EmperorOrokuSaki) [reply contracts/deployments/qanet/FleekERC721.json:1]: I don't have a strong opinion on this. Should they not be pushed?

File: serverless/src/functions/apps/handler.ts (hunk 1 above)
Comment (EmperorOrokuSaki) [reply serverless/src/functions/apps/handler.ts]: implemented

**Pull request merged**